window/src/os/wayland/frame.rs

```diff
@@ -2,6 +2,8 @@
 //! in smithay_client_toolkit 0.11 which is Copyright (c) 2018 Victor Berger
 //! and provided under the terms of the MIT license.
 
+// TODO: update this for SCTK 0.18 and use this instead of the FallbackFrame
```

Comment:
Alternatively, instead of using a built in implementation should we use [sctk-adwaita](https://crates.io/crates/sctk-adwaita) instead? Possibly as an optional dependency?